runtime/src/execute_cost_table.rs

```diff
@@ -44,7 +44,6 @@ impl ExecuteCostTable {
         self.table.len()
     }
 
-    /// default program cost, set to ComputeBudget::DEFAULT_COMPUTE_UNIT_LIMIT
```

Comment:
maybe, `DEFAULT_COMPUTE_UNIT_LIMIT` should be `DEFAULT_INSTRUCTION_COMPUTE_UNIT_LIMIT`. typo at #25229

moreover, i think this comment adds little value, so better off removing it altogether. that's because the fn body is small and the fn name alone is enough while the comment is being redundant.